web/frontend/angular/.eslintrc.json

```diff
@@ -0,0 +1,40 @@
+{
```

Comment:
You have two eslintrc files in different formats and with different content. You should only use one format and merge the content.